pkg/output/printer.go

```diff
@@ -23,7 +26,7 @@ func NewConsolePrinter() *ConsolePrinter {
 }
 
 func (c *ConsolePrinter) Printf(format string, args ...interface{}) {
-	fmt.Printf(format, args...)
+	_, _ = fmt.Fprintf(os.Stderr, format, args...)
```

Comment:
Not sure to understand why we're using stderr here

Reply:
That allows to have progressbar in the snyk CLI.

stderr should be used to print diagnostic output, by doing that we can have progressbar and other diagnostic messages displayed to the user while maintaining a clean stdout with only drift results.